controllers/admin/AdminCategoriesController.php

```diff
@@ -1003,7 +1003,9 @@ public function ajaxProcessUpdatePositions()
                     $category->cleanPositions((int) $category->id_parent);
                 }
 
-                die(true);
+                die(json_encode([
```

Comment:
Can we use ajaxRender, or ajaxDie or something else? @Quetzacoalt91

Reply:
Let's keep the die() for now, as it is already working this way.